Packs/ThreatGrid/Integrations/ThreatGrid/ThreatGrid.yml

```diff
@@ -1042,11 +1075,19 @@ script:
     outputs:
     - contextPath: Threatgrid.SearchResult
       description: Result for the searched query
-    description: Advanced search that allows searching URLS, Submissions, Samples
-      etc...
-  
+    description: Advanced search that allows searching URLS, Submissions, Samples etc...
+
+    deprecated: true
   runonce: false
   dockerimage: demisto/python3:3.10.9.42476
 tests:
-- ThreatGridTest
+- No tests (deprecated)
 fromversion: 5.0.0
+detaileddescription: |-
+  1. In your threat grid instance, click your user name at the right top corner.
+  2. Select my account.
+  3. Under the API Key section in the left you will see your API.
+
+  ---
+  [View Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/threat-grid)
```

Comment:
what is this? Does it belong to v2?

Reply:
I don't understand what you asking. This is information about ThreatGrid.

Reply:
unresolving - is this solved?

Reply:
yes